content/src/main/content/jcr_root/apps/acs-commons/components/utilities/qr-code-generator/clientlibs/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    categories="[acs-commons.qr-code-generator.app,cq.authoring.dialog]"
```

Comment:
We shouldnt include this in `cq.authoring.dialog` by default since it introduces this code "unexpectedly" into the code execution flow of the page editor. If someone doesnt want to use this feature, they still incur the cost (ex. if there is a bug we find later) of dealing w/ this feature.

It also seems weird that `cq.authoring.dialog` is the same category as the NG app exposed under etc. Shouldn't these be 2 diff clientlibs:

* acs-commons.qr-code-generator => the script.js thats used in page editor
* acs-commons.qr-code-generator.app => the NG app